syntax = "proto3";

message RequestMsg {
  int32 cmdType = 1;
  bytes clientID = 2;
  bytes serverID = 3;
  bytes sign = 4;
  bytes data = 5;
}

message ResponseMsg {
  bool status = 1;
  int32 secKeyId = 2;
  bytes clientID = 3;
  bytes serverID = 4;
  bytes data = 5;
}